### DISCUSSION TITLE
Migration from v1 (Reakit) to v2 (Ariakit) process?

### DISCUSSION THREAD

**Discussion by RockinRonE** [ariakit/ariakit#1565] [Questions] [answered]
I recently implemented Reakit at my job, but now I'm seeing that Ariakit is in alpha. Will there be a migration process, or will this be a completely new library in which existing components will need to updated with?

**Comment by diegohaz** [comment, marked as answer]
Hey! You'll find more information on [What is Ariakit?](https://gist.github.com/diegohaz/bc07491aee61a5f2469574b38c5c1aa0)

**Reply by RockinRonE** [reply]
Ah, thank you! Totally missed that. I definitely agree that v2 is already better than Reakit! Cheers!
